### PR TITLE
Feature added

### PR DESCRIPTION
**Pull request by cservan**
Add the possibility to use environment variable to launch the API

### CODE REVIEW

**Comment by cservan** [comment]
Resolve issue #21

File: src/text-classifier.cpp [hunk 1]
```diff
@@ -72,25 +73,33 @@ void ProcessArgs(int argc, char **argv) {
     }
   }
   if (model_config == "") {
```
Comment (maudetes) [inline src/text-classifier.cpp:96]: Should also take into account environment variable.

File: README.md [hunk 2]
```diff
@@ -19,9 +19,17 @@ docker build -t text-classifier:latest .
 --port (-p)              port to use (default 9009)
 --threads (-t)           number of threads (default 1)
 --debug (-d)             debug mode (default false)
+--grpc (-g)              use grpc service instead of rest
```
Comment (maudetes) [inline README.md:22]: Also add [--grpc] in .usage() in text-classifier.cpp

**Review by maudetes — COMMENTED** [review]
Some things to modify, but overall looks good to me

File: README.md (hunk 2 above)
Comment (cservan) [reply README.md:22]: Corrections asked done

**Review by maudetes — APPROVED** [review]
Seems good to me.

**Pull request merged**